Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -294,6 +298,22 @@
         'BeamSpot': 'GaussSigmaZ4cm',
         'ScenToRun' : ['GenSimFull','DigiFull','RecoFull','HARVESTFull'],
     },
+    '2023' : {
+        'Geom' : 'DB:Extended',
+        'GT' : 'auto:phase1_2023_realistic',
+        'HLTmenu': '@relval2017',
```

Comment:
Hi @zhenhu I have a question. Is there a particular reason to have a fake menu in these relvals? I see that recently a new tag `@relval2021' pointing to a real menu has been made available in https://github.com/cms-sw/cmssw/pull/27331

Reply:
@zhenhu @mmusich indeed the idea is to use the new menu for Run3 as proposed by @Martin-Grunewald in #27331 for already existing workflows